src/DataBlobs/services/BlobStores.jl

```diff
@@ -77,9 +77,13 @@ function addData!(dfg::AbstractDFG, blobstore::AbstractBlobStore, label::Symbol,
     return de=>db
 end
 
-function updateData!(dfg::AbstractDFG, blobstore::AbstractBlobStore, label::Symbol,  entry::AbstractDataEntry, blob::Vector{UInt8})
-    assertHash(entry, blob, hashfunction=hashfunction)
-    de = updateDataEntry!(dfg, label, entry)
+function updateData!(dfg::AbstractDFG, blobstore::AbstractBlobStore, label::Symbol,  entry::AbstractDataEntry, blob::Vector{UInt8}; hashfunction = sha256)
+    # Recalculate the hash - NOTE Assuming that this is going to be a BlobStoreEntry. TBD.
```

Comment:
Yes. That should be save to assume. Entry type should become ::BlobStoreEntry

Reply:
I’m not too sure about recalculating the hash though.

Reply:
Because you're updating the data itself, we need to recalculate the hash so that when you retrieve it next time, everything matches up. Or am I missing something? Will merge and we can discuss.

Reply:
I would still go with `assertHash` the entry is not created in this function, but given by the user. So the assert is there to tell the user the blob does not match the entry.